packages/analytics/client/index.ts

```diff
@@ -35,15 +35,15 @@ export type {
   ContextProviderOpts,
   // Types for the registerEventType API
   EventTypeOpts,
-} from './analytics_client';
+} from './src/analytics_client';
```

Comment:
Is there a reason we shouldn't have a main `src/index.ts` file and have the root `index.ts` re-export from that?

Reply:
@TinaHeiligers the objective of this PR was to be as less disruptive as we could make it and thats why I haven't created an index inside the `src` folder. If in your case you guys just prefer to do it that is not a problem 😃